### PR TITLE
Modifying a couple stuff to the bot since we now support 1 letter subdomains

### PR DESCRIPTION
**Pull request by iostpa**
Hope I did it right lmao

### CODE REVIEW

File: commands/check.js [hunk 1]
```diff
@@ -18,14 +18,7 @@ module.exports = {
...
         // get the guild id from the interaction
         
-
-        if (subdomain.length < 2 || subdomain.length > 64) {
-            const sadEmbed = new EmbedBuilder()
-                .setDescription("The subdomain length must be between 2 and 64 characters.")
-                .setColor("#0096ff");
-            return await interaction.reply({ embeds: [sadEmbed] });    
-        }
-
```
Comment (wdhdev) [inline commands/check.js:28]: This still needs to be added except with the limits of 1 to 100 characters.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: commands/check.js (hunk 1 above)
Comment (CuteDog5695) [reply commands/check.js]: Should be fine now

File: commands/check.js [hunk 2]
```diff
@@ -18,14 +18,13 @@ module.exports = {
             .toLowerCase();
         // get the guild id from the interaction
         
-
-        if (subdomain.length < 2 || subdomain.length > 64) {
+if (subdomain.length < 1 || subdomain.length > 64) {
```
Comment (wdhdev) [inline commands/check.js:21]:
```suggestion
if (subdomain.length < 1 || subdomain.length > 100) {
```

File: commands/check.js [hunk 3]
```diff
@@ -18,14 +18,13 @@ module.exports = {
...
             const sadEmbed = new EmbedBuilder()
-                .setDescription("The subdomain length must be between 2 and 64 characters.")
+                .setDescription("The subdomain length must be between 1 and 64 characters.")
```
Comment (wdhdev) [inline commands/check.js:23]:
```suggestion
                .setDescription("The subdomain length must be between 1 and 100 characters.")
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by iostpa** [comment]
Since you want it to be 100 characters, I'll edit the other file so that it can also support up to 100 characters

**Review by wdhdev — APPROVED** [review]

**Pull request merged**